blockly/generators/arduino/boards.js

```diff
@@ -166,6 +166,37 @@ Blockly.Arduino.Boards.profiles.leonardo = {
 /** Arduino Yun board profile is identical to Leonardo. */
 Blockly.Arduino.Boards.profiles.yun = Blockly.Arduino.Boards.profiles.leonardo;
 
+/**
+ * Arduino Nano board profile
+ * The individual nano_168p and nano_328p profiles are created
+ * because even though they have the same blockly implimentation there is
+ * a compilation distinction.
+ */
+Blockly.Arduino.Boards.profiles.nano_328p = {
+  name: 'Arduino Nano',
+  description: 'Arduino Nano with ATMega168p or ATMega328p board',
+  analogPins: Blockly.Arduino.Boards.generateAnalogIo(0,8),
```

Comment:
Looking at https://www.arduino.cc/en/Main/ArduinoBoardNano it seems that the Analogue pings go from A0 to A7, so the call should be `generateAnalogIo(0, 7)`.